dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/content/ContentMap.java

```diff
@@ -61,16 +64,18 @@
 	private Structure structure;
 	private String title;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "title" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)